src/gfx/process.cpp

```diff
@@ -953,8 +965,56 @@ static UniqueTiles dedupTiles(
 	// by caching the full tile data anyway, so we might as well.)
 	UniqueTiles tiles;
 
+	if (!options.inputTileset.empty()) {
+		File inputTileset;
+		if (!inputTileset.open(options.inputTileset, std::ios::in | std::ios::binary)) {
+			fatal("Failed to open \"%s\": %s", options.inputTileset.c_str(), strerror(errno));
+		}
+
+		std::array<uint8_t, 16> tile;
+		size_t const tileSize = options.bitDepth * 8;
```

Comment:
Hmm, so this *does* support `-d 1` and thereby allow the input tileset to be 1bpp? Then my comments above should be adapted to reflect that.

Reply:
This was explained in a comment below; thank you. <3